packages/airnode-abi/README.md

```diff
@@ -0,0 +1,71 @@
+# @airnode/airnode-abi
+
+> Encoding and decoding utilities intended for use with Airnode
+
+## Description
+
+Airnode-ABI provides a unique way to encode and decode parameters. Parameters are provided with encoding types, names and values. The types are shortened and grouped with a version as the "header". The name/value pairs are then grouped and encoded as the rest of the body.
+
+An advantage of encoding parameters this way is that parameters can be decoded natively using the language(s) of the specific blockchain. In the case of EVM blockchains, this means that parameters can be encoded as well as decoded in Solidity (or Vyper), without any additional requirements.
+
+You can find additional documentation in the [Airnode ABI specifications](https://github.com/api3dao/api3-docs/blob/master/airnode/airnode-abi-specifications.md)
+
+## Installation
+
+You can install with either [npm](https://docs.npmjs.com/getting-started/installing-node#install-npm--manage-npm-versions) or [Yarn](https://yarnpkg.com/en/docs/install)
+
+```sh
+# npm
+npm install --save @airnode/airnode-abi
+
+# Yarn
+yarn add @airnode/airnode-abi
+```
+
+## Usage
+
+### Encoding
+
+**NB:** The `types`, `names` and `values` array inputs must all have the same number of elements
```

Comment:
I thought we agreed on an array of objects instead of 3 arrays

Reply:
Ok I misunderstood the conclusion. I'll adjust